src/routes/categories/components/CategoryList.js

```diff
@@ -4,7 +4,7 @@ import React from 'react'
 
 import RemoteContent from '../../../modules/common/components/RemoteContent'
 import Caption from '../../../modules/common/components/Caption'
-import CategoryListItem from './CategoryListItem'
+import CategoryListItem from './CategoryEntry'
```

Comment:
```suggestion
import CategoryEntry from './CategoryEntry'
```

This is maybe a problem somewhere else too